server/odc-service/src/main/java/com/oceanbase/odc/service/quartz/OdcJobListener.java

```diff
@@ -92,6 +93,10 @@ public void jobToBeExecuted(JobExecutionContext context) {
         ScheduleEntity scheduleEntity =
                 scheduleRepository.findById(scheduleId)
                         .orElseThrow(() -> new NotFoundException(ResourceType.ODC_SCHEDULE, "id", scheduleId));
+        ScheduleTaskContextHolder.trace(scheduleEntity.getId(), scheduleEntity.getJobType().name(), null);
```

Comment:
why should we set these thread local? 'SchedulerFileAppender' seems like don't need any variables